src/views/Events.svelte

```diff
@@ -26,7 +27,7 @@ const logoUrl = event => event.imageFile && event.imageFile.url || ''
 let eventFilter = {}
 
 $: queryStringData = qs.parse($querystring)
-$: eventFilter = populateEventFilterFrom(queryStringData)
+$: eventFilter = populateEventFilterFrom(queryStringData, $me)
```

Comment:
after the data location of participation, I don't think you'll want to react to `me` changes...

Reply:
just to confirm, we **DO** want to react to `$me` here because of the times when someone comes straight into this page and the user response may not have returned yet, correct?

Reply:
Correct